docs/appendices/release-notes/unreleased.rst

```diff
@@ -151,4 +151,17 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
-None
+- Fixed an issue that caused ``SELECT * `` statements to fail if a table has an
+  object with inner null object and a sibling column with the same name with
+  one of the sub-columns. An example::
+
+    CREATE TABLE IF NOT EXISTS "t" (
```

Comment:
looks very verbose but with example it's easier to understand the description.

Reply:
I'm always +1 to use examples along with the description, even if the description is clear, so totally fine imho to keep it.